src/server-glue/module.ts

```diff
@@ -44,13 +44,13 @@ export const init = (
 	app: Express.Application,
 	config?: string | configLoader.Config,
 ): Bluebird<ReturnType<typeof configLoader.setup>> =>
-	sbvrUtils
-		.setup(app, db)
-		.then(async () => {
+	Bluebird.try(async () => {
```

Comment:
I had to use `Bluebird.try` instead of `Bluebird.method` because `Bluebird.method` types don't work properly with optional params